app/bundles/EmailBundle/Tests/Transport/AmazonApiTransportTest.php

```diff
@@ -0,0 +1,199 @@
++<?php
```

Comment:
This is probably a typo.

Reply:
Thanks!, artifact of not forking first and recreating the changes from my local diff.